XU-TechnicalPaper.md

```diff
@@ -265,32 +265,32 @@ Sidenote: The broadcasted orderbooks carry the fee amount of each order in a sep
+Flawed incentivisation is believed to be the main reason why other solutions for connecting exchanges didn’t find adoption yet. Some solutions even require exchanges to pay hardware or monthly fees. From an exchange's point of view, this doesn’t add up without offering something in return, since the solutions essentially offer an easier way for user funds, and thus trading revenues, to leave the platform. Exchange Union will not charge any fees and more importantly enable the exchanges to increase revenue with trading fees, see above. Additionally, the different functionaries which are rewarded with XUC are as follows:
 
 **Incentives for Developers**
 - Each pull-request to the Exchange Union github (https://github.com/ExchangeUnion) gets peer reviewed and a score by each reviewer. XUC is paid to the contributor based on that score.
 - When the pull-request makes it into the master-branch, another XUC amount gets paid to the developer
 - Reviewers get rewarded
 - Testers get rewarded, test-net time gets rewarded
 
-
 **Incentives for Exchanges**
-- In order to have sufficient interest of exchanges joining the union, especially in the beginning, the idea is to allocate a certain amount of XUC to new exchanges joining Exchange Union. These are to be distributed to an exchange's users ('airdrop'). Advantage: all users can start using XU right away, for free for a long while.
-- These XUC can only be used to pay for services on XU. Cannot be dumped. 
-- This also means *end-users have to hold XUC in order to use XU*. This is beneficial for exposure, since this is the only way end-users know about Exchange Union. Compare Binance’s BNB model here, which gives discount on fees for holders of BNB. With XUC, users can get access to new pairs and a better price. For this the XU node would have to check a particular users XUC balance.
+- As treated in chapter 3.3, the main use of XUC is to incentivize liquidity providers, which in particular targets 'large' exchanges
+- In order to further increase interest of exchanges joining Exchange Union, especially in the beginning, the idea is to allocate a certain amount of XUC to new exchanges joining Exchange Union. These are to be distributed to an exchange's users ('airdrop'). Depending on the business model of the exchange (exchange pays taker's XUC fee, or user directly), the advantage would be that all users can start using XU right away.
+- These XUC can only be used to pay for services on XU, like paying XUC to the maker. It cannot be sold on the market. 
```

Comment:
How is this enforced?

Reply:
Yes, I don’t think it’s possible to blacklist some XUC from being traded on the market.

Reply:
Worst case legally since all other ways seem impossible.